app/constants/import_constants.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ImportConstants
+  IMPORT_MAPPING = {
+    srg_id: 'SRGID',
+    stig_id: 'STIG ID',
```

Comment:
there seems to be an inconsistency between the example files where one of them is labeled `STIG ID` and others are labeled `STIGID`

Reply:
For consistency I'm going to go ahead and just rename this to STIGID since that seems more consistent with `SRGID`.